crates/runtime/tests/mongo/mod.rs

```diff
@@ -114,28 +114,25 @@ async fn mongodb_integration_test() -> Result<(), String> {
 
     test_request_context()
         .scope(async {
-            let running_container =
-                start_mongodb_docker_container(MONGODB_PORT1)
-                    .await
-                    .map_err(|e| {
-                        tracing::error!("start_mongodb_docker_container: {e}");
-                        e.to_string()
-                    })?;
-            tracing::debug!("Container started");
-            let retry_strategy = FibonacciBackoffBuilder::new().max_retries(Some(10)).build();
-            retry(retry_strategy, || async {
-                init_mongodb_db(MONGODB_PORT1)
-                    .await
-                    .map_err(|e| {
-                        tracing::error!("Failed transiently  to initialize MongoDB database: {e}");
-                        RetryError::transient(e)
-                    })
-            })
+            let running_container = start_mongodb_docker_container(MONGODB_PORT1)
```

Comment:
For some reason these and further lines were needed to be updated because of linter